assets/src/components/stopCard.tsx

```diff
@@ -91,4 +106,15 @@ const StopCard = ({
   )
 }
 
+export const AutoPanPaddingConfig = createContext<LeafletAutoPanPadding>({})
```

Comment:
suggestion: Towards your question about names, I wonder if it would be helpful to make this more generic than autoPan by making it something like `SafeZoneConfig`. Maybe then that SafeZoneConfig could easily be shared with other functions like `onFollowerUpdate`

Reply:
Agreed!
One issue with `onFollowerUpdate` is that becuase it's called within `useEffect`, we would need to `useContext` and reconstruct/curry `onFollowerUpdate` with that value before passing to the `onUpdate` prop.

Reply:
renamed in [`b8a40ca` (#1916)](https://github.com/mbta/skate/pull/1916/commits/b8a40ca3a3cd3b7393ebb87c440893ecd86d4a1f)

Reply:
Followup question: since `MapSafeArea` seems like it could generically be used for other components, and it's already exported and referenced in `mapMarkers.tsx`, would it make sense to move it to `map.tsx` or `mapPage.tsx` or some other higher-level map file, rather than staying here in `stopCard.tsx`?

Reply:
> would it make sense to move it to `map.tsx` or `mapPage.tsx` or some other higher-level map file, rather than staying here in `stopCard.tsx`?

I just found/remembered `src/contexts`, moved 👍🏻 8841cd3fd366c6eaaa3623ecc30e184b1c8408df